engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -939,10 +939,12 @@ class PlatformDispatcher {
     call `updateSemantics`.
   ''')
   void updateSemantics(SemanticsUpdate update) =>
-      _updateSemantics(update as _NativeSemanticsUpdate);
+      _updateSemantics(update as _NativeSemanticsUpdate, 0);
```

Comment:
Prefer using `_implicitViewId ` here instead of hardcoding `0` as `0` is an implementation detail that we don't want users to depend on directly.